src/wallet/wallet.ts

```diff
@@ -268,6 +268,17 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
     };
   }
 
+  /**
+   * Get server URL from config object
+   *
+   * @memberof HathorWalletServiceWallet
+   * @inner
+   */
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
It's complaining that the method doesn't use `this`, so it can't be a class method. But I need it to be like this, so both facades are equal, then I'm just disabling the rule for this method